File: MobEye/MobEye/MobEye/Requests/AuthorizationRequest.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace MobEye.Requests
{
    class AuthorizationRequest
    { 
        public String PhoneID { get; set; }
        public String PrivateKey { get; set; }

        public AuthorizationRequest(String phoneId, String privateKey)
        {
            this.PhoneID = phoneId;
            this.PrivateKey = privateKey;
        }

        public override string ToString()
        {
            return "Phone id " + PhoneID + "Private Key " + PrivateKey;
        }
    }
}

File: mobeyeSpring/mobeye_testing_api/src/main/java/com/mobeye/mobeye_testing_api/Requests/AuthorizationRequest.java
package com.mobeye.mobeye_testing_api.Requests;

public class AuthorizationRequest {
    private String phoneImei;
    private String private_key;

    public AuthorizationRequest(String phoneImei, String private_key) {
        this.phoneImei = phoneImei;
        this.private_key = private_key;
    }

    public String getPhoneImei() {
        return phoneImei;
    }

    public void setPhoneImei(String phoneImei) {
        this.phoneImei = phoneImei;
    }

    public String getPrivate_key() {
        return private_key;
    }

    public void setPrivate_key(String private_key) {
        this.private_key = private_key;
    }
}
